test_vae_training/1_train_vae.py

```diff
@@ -0,0 +1,86 @@
+# ---
+# jupyter:
+#   jupytext:
+#     formats: ipynb,py
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.9.1+dev
+#   kernelspec:
+#     display_name: Python [conda env:microbe] *
+#     language: python
+#     name: conda-env-microbe-py
+# ---
+
+# # Train VAE model
+#
+# This notebook will first try to train the current VAE model before modifying the loss function to work with count data
+
+# %load_ext autoreload
+# %autoreload 2
+# %matplotlib inline
+import os
+import matplotlib.pyplot as plt
+import pandas as pd
+from ponyo import utils, train_vae_modules
+
+# Set seeds to get reproducible VAE trained models
+train_vae_modules.set_all_seeds()
+
+# +
+base_dir = os.path.abspath(os.path.join(os.getcwd(), "../"))
+
+# Read in config variables
+config_filename = os.path.abspath(
+    os.path.join(base_dir, "test_vae_training", "config_current_vae.tsv")
+)
+
+params = utils.read_config(config_filename)
+
+dataset_name = params["dataset_name"]
+
+raw_compendium_filename = params["raw_compendium_filename"]
+normalized_compendium_filename = params["normalized_compendium_filename"]
+# -
+
+# Try normalzing the data
+train_vae_modules.normalize_expression_data(
```

Comment:
How is this being normalized? My guess is transcripts per million or percentage of total sample reads, but it would be helpful to state explicitly in the comment.

Reply:
Here I am just 0-1 normalizing each taxon. I will add a comment here.